io.github.ja2_stracciatella.JA2-Stracciatella.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.ja2_stracciatella.JA2-Stracciatella
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+command: ja2-launcher-wrapper
+rename-desktop-file: ja2-stracciatella.desktop
+rename-icon: ja2-stracciatella
+finish-args:
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
```

Comment:
We've [discussed this before](https://github.com/flathub/flathub/pull/3666#discussion_r1029655565) and I still don't understand why I have to remove it, when the app requirements page states that this permission can be used freely.

Reply:
it is not used by default. It's a sandbox. You don't make holes in the sandbox because a power user might make some changes.

Reply:
Then why does app requirements say I may use this permission _freely_?  

I'm sorry, but this again feels rather arbitrary… I mean no offence, but this comment seems to downright contradict what's documented in app requirements 🤷🏿 

Anyway, if it's all too important to you, I'm happy to remove this, as I can easily override it.  But before I do so I'd like to have clarification that the license of JA2 is acceptable for flathub, i.e. that technical issues aside, there's a perspective to have this added to flathub.

Because if there's not and I end up maintaining this outside flathub anyway, I can just as well have it my way 🤷🏿

Reply:
I'm sorry, but are you really going to insist that the usage of "freely" invalidates what you're being asked? The default `finish-args` should correspond to the real world, not some theoretical scenarios of "but what if user needs floppy to PCMCIA adapter on Thursdays". If I swap `x11` to `fallback-x11` or remove `x11` altogether, it doesn't work on my system by default **at all**. It should be removed.

Reply:
No, I'm not.  I've said that I'm happy to remove this argument, right in the comment before yours.

I'm just trying to understand by what rules and guidelines manifests are being reviewed.  This is my second pull request here, and from my point of view both reviews felt absolutely arbitrary.  

I mean no offence, again, but so far the only guiding principle I can see is "We know better so shut up" 🤷🏿